src/components/modals/login/wallets/subWallet/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Chain, Wallet } from '@rainbow-me/rainbowkit'
+import { SubWalletConnector } from '@subwallet/wagmi-connector'
+
+export interface MyWalletOptions {
```

Comment:
extract this type to maybe /wallets/type.ts ?
because this type is also used in talisman/index.ts one